scripts/tests/test_prepare_infra_k8s_inputs.py

```
@@ -0,0 +1,127 @@
+                        "max_nodes": 2,
+                    }
+                ]
+            ),
+            "INPUT_ENABLE_TRAEFIK": "false",
+            "INPUT_DRY_RUN": "true",
+        }
+    )
+    for key, value in env.items():
+        monkeypatch.setenv(key, value)
+
+    inputs = _resolve_all_inputs(RawInputs(**{field: None for field in RawInputs.__annotations__}))
```

Ruff (0.14.13)

Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable, value)`)

(C420)